### PR TITLE
0.6.0 saved synchronized

### PR DESCRIPTION
**Pull request by PiercarloSlavazza**
Addressing issue #25 .

### CODE REVIEW

**Comment by PiercarloSlavazza** [comment]
Relevant commit is obviously 9b32506

**Comment by Cretezy** [comment]
Nice work. I would actually move this up to be in `redux_persist`, so it applies to both web and Flutter.

Also you should [global ignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) all `*.iml` and `.idea` files.

File: packages/redux_persist_flutter/redux_persist_flutter.iml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Cretezy) [inline packages/redux_persist_flutter/redux_persist_flutter.iml:1]: Ignore globally (as said in thread comment) and `git rm --cached packages/redux_persist_flutter/redux_persist_flutter.iml`

File: packages/redux_persist_flutter/pubspec.yaml [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
+version: 0.6.0-save-sync-2-SNAPSHOT
 homepage: https://github.com/Cretezy/redux_persist/tree/master/packages/redux_persist_flutter
 author: Charles Crete <charles@cretezy.com>
 
 environment:
-  sdk: ">=2.0.0-dev <2.0.0"
+  sdk: ">=2.0.0-dev <3.0.0"
 
 dependencies:
+  synchronized: "^1.5.3"
```
Comment (Cretezy) [inline packages/redux_persist_flutter/pubspec.yaml:11]: Move to `redux_persist` package.

File: packages/redux_persist_flutter/pubspec.yaml [hunk 3]
```diff
@@ -1,18 +1,19 @@
 name: redux_persist_flutter
 description: Redux Persist Flutter Integration
-version: 0.6.0-rc.1
+version: 0.6.0-save-sync-2-SNAPSHOT
```
Comment (Cretezy) [inline packages/redux_persist_flutter/pubspec.yaml:3]: Bump it to `0.6.0` please

File: packages/redux_persist_flutter/pubspec.yaml [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
 homepage: https://github.com/Cretezy/redux_persist/tree/master/packages/redux_persist_flutter
 author: Charles Crete <charles@cretezy.com>
 
 environment:
-  sdk: ">=2.0.0-dev <2.0.0"
+  sdk: ">=2.0.0-dev <3.0.0"
```
Comment (Cretezy) [inline packages/redux_persist_flutter/pubspec.yaml:8]: This is odd. I remember doing this change. I might have an unpushed commit on my desktop

**Review by Cretezy — CHANGES_REQUESTED** [review]

**Comment by Cretezy** [comment]
This was merged in 0.8.0-rc.0

**Pull request closed without merging**